README.adoc

```diff
@@ -43,5 +43,13 @@ public class MyApp {
 }
 ----
 
+=== Sign the Contributor License Agreement
+Before we accept a non-trivial patch or pull request we will need you to sign the
+https://cla.pivotal.io/sign/spring[Contributor License Agreement].
+Signing the contributor's agreement does not grant anyone commit rights to the main
+repository, but it does mean that we can accept your contributions, and you will get an
+author credit if we do.  Active contributors might be asked to join the core team, and
+given the ability to merge pull requests.
+
```

Comment:
Can we break this out into a separate commit?  The addition of the CLA tool is more than just the documentation related to it...